packages/application-shell-connectors/src/components/application-context/application-context.tsx

```diff
@@ -72,6 +89,33 @@ export const mapUserToApplicationContextUser = (user?: TFetchedUser) => {
     timeZone: user.timeZone || defaultTimeZone,
     projects: user.projects,
   };
+
+  // This property will only be populated when user has logged in using SSO
+  if (user.idTokenUserInfo) {
+    let additionalClaims: Record<string, unknown> = {};
+    try {
+      additionalClaims = JSON.parse(
+        user.idTokenUserInfo.additionalClaims || ''
+      );
+    } catch (error) {
+      console.warn(
+        '@commercetools-frontend/application-shell-connectors: Could not parse received user sso custom claims from server.',
+        (error as Error).message
+      );
+    }
+    applicationContextUser.idTokenUserInfo = {
+      issuer: user.idTokenUserInfo.iss,
+      subject: user.idTokenUserInfo.sub,
+      audience: user.idTokenUserInfo.aud,
+      expirationTimestamp: user.idTokenUserInfo.exp,
+      creationTimestamp: user.idTokenUserInfo.iat,
```

Comment:
Any particular reason for renaming these fields? I would expect to expose them as they are.

Reply:
I thought that, since developers using this data would be frontend engineers, they (actually everyone) would be more comfortable with more meaningful names. I would use the abbreviations for internal usage but use something more clear in the user data model.

Perhaps I overthought it and it would be easier to just map the same attribute names we get from the IdP token.

Let me know if I should rename these props.

Reply:
I think for now I would keep the same names, as we're simply exposing the claims as-is.

Also, the idea with this new field `idTokenUserInfo` is to somehow align it with how you would expose these infos in the `/userinfo` endpoint.

Reply:
Updated in this commit: 0138b05933457a87daefeace91129884567af75a